src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -266,7 +266,13 @@ private void RunAssembly(string assemblyName, TestFilter filter)
                     }
                 }
                 else
-                    TestLog.Info("NUnit failed to load " + assemblyName);
+                {
+                    var msgNode = loadResult.SelectSingleNode("properties/property[@name='_SKIPREASON']");
+                    if (msgNode != null && (new[] { "contains no tests", "Has no TestFixtures" }).Any(msgNode.GetAttribute("value").Contains))
+                        TestLog.Info("Assembly " + assemblyName + " contains no tests supported by this runner: " + _activeRunner.GetType().AssemblyQualifiedName);
```

Comment:
Yeah, well - I don't know what that should be. I stumbled across this issue when one of the projects in my solution was linked with an older version of NUnit, I have no idea what other situations could cause this.
Of course I can change the text runner with TestRunner, but that doesn't help much.